packages/strapi-admin/services/permission/sections-builder/handlers.js

```diff
@@ -92,8 +94,8 @@ const subjectsHandlerFor = kind => ({ action, section: contentTypesSection }) =>
   contentTypesSection.subjects.push(...newSubjects);
 };
 
-const buildNode = ([attributeName, attribute]) => {
-  if (attribute.configurable === false) {
+const buildNodeFor = model => ([attributeName, attribute]) => {
```

Comment:
can you switch to using two args or an object here. this syntax is too specific to Object.entries + map